Objective-C/CBLDatabase.h

```diff
@@ -244,14 +253,23 @@ typedef NS_ENUM(uint32_t, CBLConcurrencyControl) {
  */
 - (BOOL) delete: (NSError**)error;
 
+/**
+ Performs database maintenance.
+ 
+ @param type Maintenance type.
+ @param error On return, the error if any.
+ @return True on success, false on failure.
+ */
+- (BOOL) performMaintenance: (CBLMaintenanceType)type error: (NSError**)error;
+
 /**
  Compacts the database file by deleting unused attachment files and vacuuming 
```

Comment:
" This method is deprecated. " text in the description?

Reply:
Yeah, will add that.

Reply:
Done